package.json

```diff
@@ -19,6 +20,7 @@
     "react-dom": "^16.12.0",
     "react-router-dom": "^5.1.2",
     "react-scripts": "^3.2.0",
+    "react-select": "^3.0.8",
```

Comment:
There are still references to react-select in the code, so you might want to remove those and `npm uninstall react-select @types/react-select` :)